google-cloud-spanner/src/test/java/com/google/cloud/spanner/GrpcResultSetTest.java

```diff
@@ -867,4 +886,22 @@ public void getDateList() {
     assertThat(resultSet.next()).isTrue();
     assertThat(resultSet.getDateList(0)).isEqualTo(dateList);
   }
+
+  @Test
+  public void getJsonList() {
+    List<Json> jsonList = new ArrayList<>();
+    jsonList.add(new Json("{\"color\":\"red\",\"value\":\"#f00\"}"));
+    jsonList.add(new Json("[]"));
```

Comment:
I think it would be good to also add a null value to the list to ensure that decoding and comparing null values inside of an array is also handled correctly.